#!/usr/bin/env python
PACKAGE = "turtlebot3_autorace_detect"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hue_red_l",        int_t,      0,      "hue_red_l",        0,  0, 179)
gen.add("hue_red_h",        int_t,      0,      "hue_red_h",        179,  0, 179)
gen.add("saturation_red_l",     int_t,      0,      "saturation_red_l",     0,  0, 255)
gen.add("saturation_red_h",     int_t,      0,      "saturation_red_h",     255,  0, 255)
gen.add("lightness_red_l",     int_t,      0,      "lightness_red_l",     0,  0, 255)
gen.add("lightness_red_h",     int_t,      0,      "lightness_red_h",     255,  0, 255)

exit(gen.generate(PACKAGE, "turtlebot3_autorace_detect", "DetectLevelParams"))